manim/mobject/coordinate_systems.py

```diff
@@ -119,27 +120,26 @@ def input_to_graph_point(self, x, graph):
                 return None
 
 
+@attr.s(auto_attribs=True, eq=False)
 class Axes(VGroup, CoordinateSystem):
-    CONFIG = {
-        "axis_config": {
+    axis_config: tp.Dict = {
```

Comment:
same comment about removing CONFIG later.
```suggestion
    axis_config: dict = {
```